packages/vanilla/webpack.common.js

```diff
@@ -0,0 +1,55 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'hig.js',
+    path: path.resolve(__dirname, 'lib'),
+    library: 'Hig',
+    libraryTarget: 'umd'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        use: 'babel-loader',
+        exclude: [/node_modules/]
+      },
+      {
+        test: /\.scss$/,
+        use: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: [
+            { loader: 'css-loader' },
+            { loader: 'sass-loader' },
+          ]
+        }),
+      },
+      {
+        test: /\.(html|svg)$/,
+        issuer: /\.js$/,
+        use: 'raw-loader'
+      },
+      {
+        test: /\.svg$/,
+        issuer: /\.scss$/,
+        use: 'svg-url-loader'
+      }
+    ]
+  },
+  resolve: {
+    alias: {
+      'interface.json': 'hig-interface',
+      '_core.js': path.resolve(__dirname, 'src/helpers/js/_core.js')
+    },
+    extensions: ['.js', '.json'],
+    modules: [
+      path.resolve(__dirname, 'src'),
```

Comment:
This one-liner is the magic that allows you to import from `hig/packages/vanilla/src`.

Reply:
@nfiniteset is there a reason that we weren't doing this before?

Reply:
I also found some old aliases that might have been an attempt to get this working previously

https://github.com/Autodesk/hig/pull/530/files#diff-a1252716d499097ecad8794c537f675eL51

Reply:
that's possible